apps/examples/testcase/le_tc/kernel/tc_libc_spawn.c

```diff
@@ -43,25 +43,25 @@
 #define SPAWN_STACKSIZE 2048
 
 /**
-* @fn                   :function_name_spawn
-* @description          :Function used for task_spawn
-* @return               :int
-*/
+ * @fn                   :function_name_spawn
```

Comment:
Don't change another kind of change in same commit.
Split formatting and adding new TCs.
Please mind this.